session/sessionstate.go

```diff
@@ -469,6 +469,17 @@ func (state *State) RegisterEvent(handle int,
 	})
 }
 
+// GetEvent for handle
+func (state *State) GetEventFunc(handle int) func(ctx context.Context, actionState *action.State) error {
+	state.eventMu.Lock()
+	defer state.eventMu.Unlock()
+	event, exist := state.events[handle]
+	if exist && event != nil {
+		return event.F
+	}
```

Comment:
minor: Could be simplified
```suggestion
	event := state.events[handle]
	if event != nil {
		return event.F
	}
```

Reply:
having the `exists` helps if you ever need to debug, to distinguish between, exists with value null and doesn't exist.